UPGRADE-5.0.md

```diff
@@ -0,0 +1,24 @@
+UPGRADE FROM 4.x to 5.0
+=======================
+
+Database
+------
+
+Many LanSuite installations have database table prefixes.
+We assume that your database table prefix is `ls_`.
+If this is different for your installation, please change the table names in the queries accordingly.
+
+ * The column `cfg_key` in table `config_selections` was extended from `CHAR(16)` to `VARCHAR(50)`
+
+   To upgrade, execute the query
+
+   ```sql
+   ALTER TABLE `ls_config_selections` MODIFY `cfg_key` VARCHAR(50);
```

Comment:
Changes to the DB structure based on the XML definition can be automatically applied with the DB manager.

Reply:
True.
I thought this might be useful for people who modified their instance. But i might think to much.
Do you still think this file make sense? If yes, we can describe this here, that the changes can be done via the Admin installation module.
Or maybe useful for people who wrote their own modules.
Or is this too much? Whats your suggestion? Removing or keeping this file? And when keeping, which purpose should this fulfill?

Reply:
A general update documentation(per release) is a good idea, also started one in the wiki a while ago:
https://github.com/lansuite/lansuite/wiki/HowTo%3A-Upgrade-from-Release-4.2
I would only document the queries that cannot be covered by the existing db update (e.g. necessary DML statements or preparative / rollback steps), because the rest can be seen either by a diff of the XML documents or by just looking at the generated DDL statements on execution

Reply:
Fine for me. Removed the file again and created https://github.com/lansuite/lansuite/issues/214.
Feel free to continue with the review.